UnityProject/Assets/Scripts/Items/Medical/Genetics/MutationGrabber.cs

```diff
@@ -0,0 +1,108 @@
+using System.Collections;
+using System.Collections.Generic;
+using Systems.MobAIs;
+using UnityEngine;
+
+public class MutationGrabber : MonoBehaviour, IExaminable , ICheckedInteractable<PositionalHandApply>
+{
+
+	public List<MutationSO> CarryingMutations;
```

Comment:
`public List<MutationSO> CarryingMutations = new List<MutationSO>();`

Reply:
technically unity does a automatic   new List<MutationSO>(); . But it's good to add